src/main/java/nextstep/ladder/view/InputView.java

```diff
@@ -0,0 +1,31 @@
+package nextstep.ladder.view;
+
+import java.util.Scanner;
+import java.util.stream.Stream;
+
+public class InputView {
+    Scanner sc = new Scanner(System.in);
+    
+    public String[] getMembers() {
+        System.out.println("참여할 사람 이름을 입력하세요. (이름은 쉼표(,)로 구분하세요)");
+        String[] members = sc.nextLine().split(",");
+        validateNameLength(members);
+        
+        return members;
+    }
+
+    public int getLadderHeight() {
+        System.out.println("최대 사다리 높이는 몇 개인가요?");
+        int ladder = sc.nextInt();
+        
+        return ladder;
+    }
```

Comment:
```suggestion
    public int getLadderHeight() {
        System.out.println("최대 사다리 높이는 몇 개인가요?");
        return sc.nextInt();
    }
```
바로 반환해줘도 좋을 것 같아요! 😄